explorer/src/serializers/chainlinkNodesSerializer.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Serializer as JSONAPISerializer,
+  SerializerOptions,
+} from 'jsonapi-serializer'
+import { ChainlinkNode } from '../entity/ChainlinkNode'
+import { BASE_ATTRIBUTES } from './chainlinkNodeSerializer'
+
+const chainlinkNodesSerializer = (
```

Comment:
I feel like this should be attached as a `toJSON` method on the models we want to serialize. `JSON.stringify` automatically calls this method when used on an object that has it defined.

Reply:
We should have serialization tests too.